src/expr.rs

```diff
@@ -812,9 +821,9 @@ fn rewrite_if_else(context: &RewriteContext,
         try_opt!(write!(&mut result,
                         "{}else{}",
                         between_if_else_block_comment.as_ref()
```

Comment:
I think the second link (`bar`) is more easily overlooked with that style. With the one link per line style, it's clearer to find each subsequent link and your eye doesn't need to make the jump.

Pathological example:
```rust
very_long_function(with.loads().of_arguments, arg2, arg3, sub_funk(test)).foo
    .bar(xs)
```
vs
```rust
very_long_function(with.loads().of_arguments, arg2, arg3, sub_funk(test))
    .foo
    .bar(xs)
```